net-p2p/cardano-db-sync/Makefile

```diff
@@ -0,0 +1,78 @@
+PORTNAME=	cardano-db-sync
+PORTVERSION=	13.0.1
+CATEGORIES=	net-p2p
+EXTRACT_ONLY+=	input-output-hk-libsodium-${LIBSODIUM_HASH}_GH0${EXTRACT_SUFX} \
+		bitcoin-core-secp256k1-${SECP256K1_HASH}_GH0${EXTRACT_SUFX}
+
+MAINTAINER=	arrowd@FreeBSD.org
+COMMENT=	PostgreSQL-driven Cardano blockchain data tool
+		
+LICENSE=	APACHE20
+
+BUILD_DEPENDS=	bash:shells/bash \
+		git:devel/git \
+		ghc-8.10.7:lang/ghc810 \
+		postgresql14-client>0:databases/postgresql14-client
+		
+RUN_DEPENDS=	postgres:databases/postgresql14-server \
+		cardano-node:net-p2p/cardano-node
+
+LIB_DEPENDS=	libpq.so.5:databases/postgresql14-client
```

Comment:
I did see something about the libpq being needed "twice". I guess this is because i'm using USES=pgsql:14 + LIB_DEPENDS ... let me try to remove and test.

Reply:
Removed LIB_DEPENDS  build / install working.